docs/components/RadioGroup/Web.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { RadioGroup, RadioOption } from "@jobber/components/RadioGroup";
+import { Content } from "@jobber/components/Content";
+import { Divider } from "@jobber/components/Divider";
+import { Checkbox } from "@jobber/components/Checkbox";
+import { Avatar } from "@jobber/components/Avatar";
+import { Text } from "@jobber/components/Text";
+
+export default {
+  title: "Components/Forms and Inputs/RadioGroup/Web",
+  component: RadioGroup,
+  subcomponents: { RadioOption },
+  parameters: {
+    viewMode: "story",
+  },
+} as ComponentMeta<typeof RadioGroup>;
+
+const BasicTemplate: ComponentStory<typeof RadioGroup> = args => {
+  const [company, setCompany] = useState("apple");
+  return (
+    <RadioGroup
+      {...args}
+      onChange={(value: string) => setCompany(value)}
+      value={company}
+      ariaLabel="Companies"
+    >
+      <RadioOption value="apple" label="Apple" />
+      <RadioOption value="google" label="Google" />
+      <RadioOption value="microsoft" label="Microsoft" />
+    </RadioGroup>
+  );
+};
+
+const DisabledTemplate: ComponentStory<typeof RadioGroup> = () => {
```

Comment:
I'm curious, is there any reason you didn't include the `args` here for this and following examples?

Reply:
Added it, I didn't add because I was not passing any arguments, but I think it's good to be working on the canvas.